### DISCUSSION TITLE
Proxy Variable for License.exe

### DISCUSSION THREAD

**Discussion by mitchell-tesch** [idea-statica/ideastatica-public#361] [Help Q&A]
Hi IDEA.StatiCa,

Is there a configuration file or a specific System Environmental Variable I can edit to automate the use of a Proxy address by `License.exe`?

I ask because I regularly swap between in office working (on Proxy) and remote working (off Proxy). When switching between I have  the setting of the `http_proxy` and `https_proxy` system environmental variables, but these do not appear to be referenced by `License.exe`.

Kind regards,

Mitch

**Comment by dr13377** [comment]
Hi,
I have a similar issue to the above. Is there a solution to this? When working in the office the proxy is turned on and when working remotely the proxy is turned off. Is there a quick way to change whether Idea Statica is using a proxy? Currently I have to open the License.exe and sign out to bring up the option to enter or delete the proxy settings, then sign back in and open Idea Statica.
![image](https://github.com/idea-statica/ideastatica-public/assets/91706141/45fed8ea-d8dd-4004-b7c2-caaf94b09c9b)
Thanks.

**Comment by nathanlukeIDEA** [comment]
hi @mitchell-tesch  and @dr13377, sorry for the slow reply on this. We are looking into this and will get back to you.

**Reply by mitchell-tesch** [reply]
@nathanlukeIDEA  - I was wondering if you have had a chance to look into the above?

**Reply by nathanlukeIDEA** [reply]
Hi @mitchell-tesch, I have passed the request onto the development team which works on the licensing. They are processing. If there is anyone else who requires this than feel free to also comment below.

**Reply by mitchell-tesch** [reply]
Hi @nathanlukeIDEA hope you are well.
Have you heard back from the development team?

**Reply by nathanlukeIDEA** [reply]
Hi @mitchell-tesch, it is in backlog; however, I can not provide a timeframe on this. For others wanting this, I would suggest going through normal channels of support as I will close this conversation.
